src/vae.py

```diff
@@ -162,12 +161,12 @@ def Q(self, x_hat):
         # Q is the latent variables' PDF
 
         x_hat = x_hat.view(-1, self.q_in)
-        mu, log_sigma = self.q_mu(x_hat), self.q_vars(x_hat)
-        return mu, log_sigma
+        mu, log_sigma_sq = self.q_mu(x_hat), self.q_vars(x_hat)
```

Comment:
Did you change it in the other places too? I think we should change it in some other places as well to avoid misunderstandings